style.css

```diff
@@ -412,12 +412,111 @@ header {
 }
 
 .skills {
-  /* margin-left: 145px;
-  margin-right: 115px; */
   padding-right: 40px;
   margin-top: 12px;
 }
 
 .frameworks {
   padding-left: 12px;
 }
+
+.project1:hover,
+.project2:hover {
+  background-color: #daa884;
+  cursor: pointer;
+}
+
+#resume:hover {
+  background-color: #daa884;
+  cursor: pointer;
+}
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+}
+
+.contact h1 {
+  font-family: 'Inter', sans-serif;
+  font-weight: bolder;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 120px 24px 105px;
+  color: #172b4d;
+  text-align: center;
+  min-height: 216px;
+  justify-content: center;
+}
+
+.form-1 {
+  display: flex;
+  flex-direction: column;
+  margin-left: 1.5rem;
+  margin-right: 1.5rem;
+}
+
+#name,
+#email,
+#textarea-1 {
+  padding: 12px;
+  border: none;
+  margin-bottom: 14px;
+  margin-top: 39px;
+}
+
+#textarea-1 {
+  background-color: #fbf8f7;
+  color: black;
+}
+
+input::placeholder {
+  color: #979493;
+}
+
+#textarea-1::placeholder {
+  color: black;
+  font-weight: 400;
+}
+
+#get-in-touch {
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  background-color: #ff6b00;
+  color: white;
+  margin: 36px 134px 134px;
+  padding: 12px;
+  border: none;
+}
```

Comment:
Instead of setting the `margin-left`, `margin-right` and` margin-bottom` of the button to `134px`, I suggest you use `auto` for both the left and right values as it will center your button horizontally, and 124px for the bottom value :` margin: 36px auto 124px;`

### The template: 
![Screenshot from 2023-01-25 21-05-19](https://user-images.githubusercontent.com/93520178/214677510-e85a0918-2166-44b8-8160-96fe2132e13f.png)
### Your design: 
![Screenshot from 2023-01-25 21-04-43](https://user-images.githubusercontent.com/93520178/214677551-6ae71bcd-195d-4eb0-88ef-4f0ee80bfb5f.png)